fee_allocator/constants.py

```diff
@@ -0,0 +1,16 @@
+FEE_CONSTANTS_URL = "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/main/config/protocol_fees_constants.json"
+CORE_POOLS_URL = "https://raw.githubusercontent.com/BalancerMaxis/bal_addresses/main/outputs/core_pools.json"
+REROUTE_CONFIG_URL = "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/main/config/core_pools_rerouting.json"
+POOL_OVERRIDES_URL = "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/ae9cfa3d627dfb66d1fba6f824c316e4769bbf5a/config/pool_incentives_overrides.json"
```

Comment:
Why is there a hard coded commit hash here?

Reply:
nice catch. originally its because the multisig override config pe wasnt merged, so i had to reference the config from the pr in the meantime in order to test but never got around to updating it after it was merged, see: https://github.com/BalancerMaxis/multisig-ops/pull/1166

Reply:
fixed in [d2bf119](https://github.com/BalancerMaxis/protocol_fee_allocator_v2/pull/1/commits/d2bf119bbea828e2fc7cd318f267ac618ede8669)